pkg/commands/loaders/branches.go

```diff
@@ -122,13 +128,18 @@ func (self *BranchLoader) obtainBranches() []*models.Branch {
 		}
 
 		split := strings.Split(line, "\x00")
-		if len(split) != 4 {
+		if len(split) != 5 {
 			// Ignore line if it isn't separated into 4 parts
 			// This is probably a warning message, for more info see:
 			// https://github.com/jesseduffield/lazygit/issues/1385#issuecomment-885580439
 			return nil, false
 		}
 
+		if len(split[4]) > 0 && split[4] != currentDir {
```

Comment:
Can `currentDir` ever be `""`? If it can't the first condition isn't needed.

Reply:
Yup, sounds right.

Reply:
Actually, we need that check because if a branch is not checked out, then it will not have a path.